pyproject.toml

```diff
@@ -32,21 +32,21 @@ pydantic = "^1.10.4"
 orjson = "^3.8.5"
 numpy = "*"
 toml = "^0.10.2"
-nibabel = {version = "^5.0.0", extras = ["medical"], python = ">=3.8.1"}
-connected-components-3d = {version = "^3.10.3", extras = ["medical"]}
-scikit-learn = {version = "^1.2.0", extras = ["ml", "test"], python = ">=3.8.1,<3.11"}
-torch = {version = "^1.13.1", extras = ["ml"]}
-torchvision = {version = "^0.14.1", extras = ["ml"]}
-black = {version = "^22.12.0", extras = ["dev"]}
-isort = {version = "^5.11.4", extras = ["dev"]}
-flake8 = {version = "^6.0.0", extras = ["dev"], python = ">=3.8.1,<3.11"}
-mypy = {version = "^0.991", extras = ["dev"]}
-responses = {version = "^0.22.0", extras = ["test", "dev"]}
-pytest = {version = "^7.2.1", extras = ["test", "dev"]}
-pytest-describe = {version = "^2.0.1", extras = ["test", "dev"]}
-flake8-pyproject = {version = "^1.2.2", extras = ["test", "dev"], python = ">=3.8.1,<3.11"}
-debugpy = {version = "^1.6.5", extras = ["dev"]}
-types-requests = {version = "^2.28.11.8", extras = ["dev"]}
+nibabel = {version = "^5.0.0", python = ">=3.8.1"}
```

Comment:
the `extras` fields have been removed as they were referring to our extras groups, but actually causing this to try to install extras groups _on the dependencies_.

This will remove several of the confusing errors users have reported to CX